README.md

```diff
@@ -240,22 +240,22 @@ We currently have the following levels of access
 
      * Example maintainers are granted elevated access to these clusters in order to facilitate development of these tests
 
-### No results show up in Gubernator
+### No results show up in Spyglass
 
-If no results show up in Gubernator this means the prow job didn't get far enough to upload any results/logs to GCS.
+If no results show up in Spyglass this means the prow job didn't get far enough to upload any results/logs to GCS.
 
 To debug this you need the pod logs. You can access the pod logs via the build log link for your job in the [prow jobs UI](https://prow.k8s.io/)
 
   * Pod logs are ephmeral so you need to check shortly after your job runs.
 
 The pod logs are available in StackDriver but only the Google Kubeflow Team has access
-  * Prow runs on a cluster owned by the K8s team not Kubeflow
-  * This policy is determined by K8s not Kubeflow
-  * This could potentially be fixed by using our own prow build cluster [issue#32](https://github.com/kubeflow/testing/issues/32)
+  * Prow controllers run on a cluster (`k8s-prow/prow`) owned by the K8s team
+  * Prow jobs (i.e. pods) run on a build cluster (`kubeflow-ci/kubeflow-testing`) owned by the Kubeflow team
+  * This policy for controller logs is owned by K8s, while the policy for job logs is governed by Kubeflow
 
 To access the stackdriver logs 
   
-  * Open stackdriver for project [k8s-prow-builds](https://console.cloud.google.com/logs/viewer?organizationId=433637338589&project=k8s-prow-builds&folder&minLogLevel=0&expandAll=false&timestamp=2018-05-22T17:09:26.625000000Z&customFacets&limitCustomFacetWidth=true&dateRangeStart=2018-05-22T11:09:27.032Z&dateRangeEnd=2018-05-22T17:09:27.032Z&interval=PT6H&resource=gce_firewall_rule&scrollTimestamp=2018-05-22T15:40:23.000000000Z&advancedFilter=resource.type%3D"container"%0Aresource.labels.pod_id%3D"15f5a424-5dd6-11e8-826c-0a580a6c0117"%0A)
+  * Open stackdriver for project [k8s-prow-builds](https://console.cloud.google.com/logs/viewer?organizationId=433637338589&project=kubeflow-testing&folder&minLogLevel=0&expandAll=false&timestamp=2018-05-22T17:09:26.625000000Z&customFacets&limitCustomFacetWidth=true&dateRangeStart=2018-05-22T11:09:27.032Z&dateRangeEnd=2018-05-22T17:09:27.032Z&interval=PT6H&resource=gce_firewall_rule&scrollTimestamp=2018-05-22T15:40:23.000000000Z&advancedFilter=resource.type%3D"container"%0Aresource.labels.pod_id%3D"15f5a424-5dd6-11e8-826c-0a580a6c0117"%0A)
```

Comment:
Can someone please confirm this updated link is accurate (I doesn't appear that I have the correct perms)?